packages/flutter_tools/test/src/fake_process_manager.dart

```diff
@@ -118,16 +118,12 @@ class _FakeProcess implements Process {
   _FakeProcess(
     this._exitCode,
     Duration duration,
-    VoidCallback onRun,
```

Comment:
What's the reason for these updates?

Reply:
That, I don't know. This change was copied from Jonah's original CL: https://github.com/flutter/flutter/pull/65137. I'll defer to your judgement whether it should be omitted or not.